README.md

```diff
@@ -2,5 +2,12 @@
 
 [![Sugaroid Bot](https://img.shields.io/badge/sugaroid-discord-%23f70049)](https://github.com/sugaroidbot/sugaroid)
 
+Sugaroid also has a discord instance running on a ``Heroku`` server.
```

Comment:
Why 'also'? 

When you are copying something, make sure you have perfection

Reply:
Oops! sorry for that 😁